### ISSUE TITLE
Gradle plugin Icon

### ISSUE THREAD

**Issue by rajmahendra** [kelemen/netbeans-gradle-project#169]
When i use more Gradle files in the project its too much Green color allover the project. :)

For the Gradle project icon we can use the default java project icon and use gradle small icon on top of it or so like maven java project icon of netbeans. It will reduce some green color in project

**Comment by WillBDaniels** [comment]
Perhaps it would be possible to add an option under "misc" to upload a custom icon for use? Although I agree with you that the green is nice, I guess I could see it being a little overwhelming (especially if you have many multiple sub-projects, all the identical icon stacking could become cumbersome

**Comment by JustGregory** [comment]
I ... think I see what you mean, although I still like the project icon representative of Gradle itself. I've got a top-level Gradle project with around ~40 subprojects. Yes, it is a bit of a "sea of green" for the project icon in the Projects window. It's not so confusing on my end, mostly because of the project naming convention I'm using, but I can see a reason that a little variation could be useful.

Maybe it could be like how NetBean Platform projects have a different color for the Platform's Module Suite project and the Module projects, perhaps we could have a slightly different icon _color_ for the outer and inner projects; this could be as simple as a darker green for the parent project, and a lighter colored green for the inner projects. As well as maybe icon-badging for specialization of settings files, build files, and various odds-and-ends miscellaneous gradle files...

Or am I misunderstanding? The actual gradle script files, lowercase 'g', are in my mind more identified with Gradle, capital 'g'; this NetBeans plugin uses the same icon as for the scripts. And as such, perhaps we could adopt an icon that's not the same as, but similar in nature to, the logo for Gradle itself? For me, it's purely an aesthetics choice, and as long as the actual /project/ works as expected....

**Comment by rajmahendra** [comment]
My suggestions
*Top level Maven parent project  - Use same logo (Maven project shows as Me )
* Sub project -  brown java coffee cup with small current logo on top right 
* subproject folder - yellow folder with small current logo in bottom right side 
* Standalone gradle project - brown java coffee cup with small current logo on top right 

This will give a easy to identify then green domination and follows NB standard icon

**Comment by kelemen** [comment]
I agree that there should be some distincion in icons. That is, "buildSrc", the root projects (and there might be some other conditions) should have different icons. However, I'm not very good at designing reasonable icons. If someone would create some, I would glady change the icons (or maybe, even make it configurable).

**Comment by imartinbe** [comment]
It could be good to have a project icon different depending on the first language plugin applied : 
- base (gradle)
- java (gradle with cofee cup)
- war (gradle with world map)
etc...
